iib/web/models.py

```diff
@@ -619,7 +667,7 @@ def request_states(self):
         return [RequestStateMapping(request.state.state).name for request in requests]
 
     @property
-    def user(self):
+    def user(self) -> 'Optional[User]':
```

Comment:
As you've set `from __future__ import annotations` you can change this to:

```suggestion
    def user(self) -> Optional[User]:
```